s3-uploads.php

```diff
@@ -18,7 +18,7 @@ function s3_uploads_init() {
 	// Ensure the AWS SDK can be loaded.
 	if ( ! class_exists( '\\Aws\\S3\\S3Client' ) ) {
```

Comment:
I feel like we should maybe change this to instead be `if file_exists vendor/autoload` instead now?

Reply:
Hmm I think it's better the way it already was - including the autoload file will cause issues is `S3Client` is already loaded via another autoload (I think).

Reply:
this is causing issues as it thows a fatal error stating `No such file or directory in /wp-content/plugins/S3-Uploads-2.2.1/vendor/autoload.php`.

Reply:
@tylerlindell You need to use Composer to install the dependencies when updating. (Also, please file a new issue rather than commenting on old PRs, otherwise we can easily miss it!)